diffpy/pdfmorph/pdfplot.py

```diff
@@ -133,32 +138,32 @@ def comparePDFs(pairlist, labels=None, rmin = None, rmax = None, show = True,
 
     # Scale the x-limit based on the r-extent of the signal. This gives a nice
     # density of PDF peaks.
-    rlim = rvmax - rvmin
-    scale = rlim / 25.0
+    #rlim = rvmax - rvmin
```

Comment:
are you sure you need to remove these?  I am not 100% sure but don't they scale the axes to the range of the data, something like that?  Just check.

Go ahead and delete all the commented lines when you are sure they are not needed.  Do this on a single commit (i.e. there are no other changes in that commit) which would help us to get them back if we needed them again for some reason.